packages/zod/jest.config.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable */
+export default {
```

Comment:
😢 sorry for that, during this PR development I have replaces jest by vitetest. Could you migrate your package as well?